app/models/question.rb

```diff
@@ -36,4 +38,8 @@ def total_rates
   def like_ratio
     total_rates == 0 ? 0.to_f : positive_rates.to_f / total_rates
   end
+
+  def convert_correct_answers
```

Comment:
I can't figure out the purpose of this method.
I bit of comment could help:
- why this method?
- who will use it? (a controller? which one actually?)

Often, when trying to write explanatory comments, I realise my method was misnamed, of misplaced. :P